### PR TITLE
Batch update TiKV docs

### PR DESCRIPTION
**Pull request by TomShawn**
Signed-off-by: TomShawn <41534398+TomShawn@users.noreply.github.com>

### What is changed?
Batch update TiKV docs

### CODE REVIEW

File: content/docs/5.1/develop/clients/cpp.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ menu:
...
-TiKV client for C++ is built on top of [TiKV Client in Rust](https://github.com/tikv/client-rust) via [cxx](https://github.com/dtolnay/cxx).
-
-This client is still in the stage of prove-of-concept and under heavy development. You can track development at [tikv/client-cpp](https://github.com/tikv/client-cpp/) repository.
-
 {{< warning >}}
-You are not suggested to use the C++ client for production use until it is released.
+Currently, the TiKV client for C++ is not released yet.
 {{< /warning >}}
+
+The TiKV client in C++ is built on top of [TiKV client in Rust](https://github.com/tikv/client-rust) using [cxx](https://github.com/dtolnay/cxx).
```
Comment (TomShawn) [inline content/docs/5.1/develop/clients/cpp.md:14]:
```suggestion
The TiKV client for C++ is built on top of [TiKV client for Rust](https://github.com/tikv/client-rust) using [cxx](https://github.com/dtolnay/cxx).
```

File: content/docs/5.1/develop/rawkv/ttl.md [hunk 2]
```diff
@@ -33,44 +46,46 @@ TiConfiguration conf = TiConfiguration.createRawDefault("127.0.0.1:2379");
...
 client.close();
 session.close();
 ```
 
-`TTL` is also supported in the `CAS` API. Let's see an example.
+## Set TTL in the `CAS` API
+
+You can also set TTL via the `CAS` API. See the following example:
 
 ```java
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/ttl.md:90]: @marsishandsome Could you help explain what the following code does? Similar to L31 - L36.

File: content/docs/5.1/develop/clients/cpp.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ menu:
...
-This client is still in the stage of prove-of-concept and under heavy development. You can track development at [tikv/client-cpp](https://github.com/tikv/client-cpp/) repository.
-
 {{< warning >}}
-You are not suggested to use the C++ client for production use until it is released.
+Currently, the TiKV client for C++ is not released yet.
 {{< /warning >}}
+
+The TiKV client for C++ is built on top of [TiKV client for Rust](https://github.com/tikv/client-rust) using [cxx](https://github.com/dtolnay/cxx).
+
+This C++ client is still in the stage of prove-of-concept and under heavy development. You can track the development progress at the [tikv/client-cpp](https://github.com/tikv/client-cpp/) repository.
```
Comment (Joyinqin) [inline content/docs/5.1/develop/clients/cpp.md:16]:
```suggestion
This C++ client is still in the stage of proof-of-concept and under development. You can track the development progress at the [tikv/client-cpp](https://github.com/tikv/client-cpp/) repository.
```

File: content/docs/5.1/develop/rawkv/ttl.md [hunk 4]
```diff
@@ -1,26 +1,39 @@
...
 
 ```yaml
 [storage]
 enable-ttl = true
 ```
 
-## Java
+## Use TTL in Java client
+
+After TTL is enabled in TiKV, you can set it in Java client via the `put` API or `CAS` API. The following two examples shows how to set TTL via the `put` API and `CAS` API.
```
Comment (Joyinqin) [inline content/docs/5.1/develop/rawkv/ttl.md:27]:
```suggestion
After TTL is enabled in TiKV, you can set it in Java client via the `put` API or `CAS` API. The following two examples show how to set TTL via the `put` API and `CAS` API.
```

File: content/docs/5.1/reference/TiUP.md [hunk 5]
```diff
@@ -7,6 +7,6 @@ menu:
         weight: 3
 ---
 
-Starting with TiKV 4.0, TiUP, as the package manager, makes it far easier to manage different cluster components in the TiKV ecosystem.
+Starting with TiKV 4.0, TiUP, as the package manager, makes it easier to manage different cluster components in the TiKV ecosystem.
 
-Please refer to [TiUP documentation guide](https://docs.pingcap.com/tidb/stable/tiup-documentation-guide) to find more TiUP commands and usages.
+Refer to the [TiUP documentation guide](https://docs.pingcap.com/tidb/stable/tiup-documentation-guide) to find more TiUP commands and usages.
```
Comment (Joyinqin) [inline content/docs/5.1/reference/TiUP.md:12]:
```suggestion
For more TiUP commands and usages, refer to the [TiUP documentation guide](https://docs.pingcap.com/tidb/stable/tiup-documentation-guide).
```

**Review by Joyinqin — APPROVED** [review]
rest lgtm~

File: content/docs/5.1/develop/rawkv/ttl.md (hunk 2 above)
Comment (marsishandsome) [reply content/docs/5.1/develop/rawkv/ttl.md:90]: 上面的例子是单独使用 ttl，这里的例子是 ttl 配合 cas 一起使用

**Comment by marsishandsome** [comment]
@TomShawn please add sigh-off

**Comment by TomShawn** [comment]
> @TomShawn please add sigh-off

Done. PTAL again~

**Review by marsishandsome — APPROVED** [review]
LGTM

**Pull request merged**